pkg/reconciler/kubernetes/kubeclient/kubeclient.go

```diff
@@ -24,6 +25,15 @@ import (
 	"k8s.io/kubectl/pkg/util"
 )
 
+type Metadata struct {
```

Comment:
Seems we are nowhere using it.. think we can drop it completely.

Reply:
As discussed, it could be required for coming unit-tests. So we leave it for now - but if we also have late no use-case for it we should remove such potentially dead code.